lib/hydra/works/models/concerns/generic_file/contained_files.rb

```diff
@@ -10,6 +10,9 @@ module Hydra::Works::GenericFile::ContainedFiles
     directly_contains_one :original_file, through: :files, type: ::RDF::URI("http://pcdm.org/use#OriginalFile"), class_name: "Hydra::PCDM::File"
     directly_contains_one :thumbnail, through: :files, type: ::RDF::URI("http://pcdm.org/use#ThumbnailImage"), class_name: "Hydra::PCDM::File"
     directly_contains_one :extracted_text, through: :files, type: ::RDF::URI("http://pcdm.org/use#ExtractedText"), class_name: "Hydra::PCDM::File"
+    directly_contains_one :mp3, through: :files, type: ::RDF::URI("http://pcdm.org/use#ServiceFile"), class_name: "Hydra::PCDM::File"
```

Comment:
Seems like the ogg, mp3, webm, and mp4 derivatives might be better and an example of a application specific Concern name something like "ContainedFilesExtras" or "EvenMoreContainedFiles".

I like them as examples of common use cases for extending the contained files.